src/middlewares/generateDocs.ts

```diff
@@ -0,0 +1,36 @@
+import path from 'path'
+import { logInfoMessage, logLineWithIcon } from '../lib/logger.js'
+import Generator from '@asyncapi/generator'
+
+export default async (spec, config, resDir) => {
+  logInfoMessage(`Generating docs for your parsed specification...`)
+  const configData = config.generator
+  const resolvedData = spec._json
+  const generator = new Generator(
+    configData && configData.template
+      ? `@asyncapi/${configData.template}`
+      : '@asyncapi/markdown-template',
+    path.resolve(
+      resDir ? resDir : './',
+      configData && configData.folder ? configData.folder : 'docs'
+    )
+  )
+  try {
+    await generator.generateFromString(JSON.stringify(resolvedData))
+    logLineWithIcon(
+      ':zap:',
+      'Successfully generated docs for your specification'
+    )
+    return 'done'
+  } catch (error) {
+    logLineWithIcon(
```

Comment:
You can use this instead: https://github.com/asyncapi/glee/blob/master/src/lib/logger.ts#L123.

Reply:
@fmvilas I just updated the PR. Kindly review it :)